lighthouse-core/report/report-generator.js

```diff
@@ -261,14 +262,16 @@ class ReportGenerator {
     });
 
     const template = Handlebars.compile(this.getReportTemplate());
+    reportContext = reportContext || 'extension'; // could be: devtools, extension
```

Comment:
(maybe `reportContext` means that already, but without context I was thinking it was more "what generated this report" rather than "what's managing the viewer for this report")